metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "katello-katello",
-  "version": "2.0.2",
+  "version": "2.0.3",
```

Comment:
The patch level was already pre-bumped, it works like we do with tito (2.0.1 is the version of the forge).

So maybe we can leave it as 2.0.2, I guess I was wondering what the semver way to handle changing dependencies, any thoughts there? I guess since the signature of puppet-katello didn't change patch only is fine.